src/react_native/reanimated.cljs

```diff
@@ -38,7 +40,20 @@
 ;; Animated Components
 (def create-animated-component (comp reagent/adapt-react-class (.-createAnimatedComponent reanimated)))
 
-(def view (reagent/adapt-react-class (.-View reanimated)))
+(def ^:private view* (reagent/adapt-react-class (.-View reanimated)))
+
+(defn view
```

Comment:
as I mentioned before, I just wrapped the function using `time` and returning a JS Object so that reagent doesn't do any conversion on it.

I can't compare the actual work of reagent because I can't wrap its functions (or the whole process) in a `time` call.

Then you might ask, why I'm saying it takes the same amout of time... well, I checked the reagent code to transform and used the same functions -sligthly modified-, also returned a JS object with ready-to-use styles.

Reply:
@ulisesmac, the results from `time` or even https://cljs.github.io/api/cljs.core/simple-benchmark shouldn't be trusted for benchmarks. In Clojure JVM we use Criterium, which unfortunately can't be used in CLJS. If we need to benchmark lower-level stuff like what this PR is doing, then I think something of the caliber of https://github.com/bestiejs/benchmark.js should be used to help find the fastest implementations. Obviously, profiling is important too, otherwise we might be optimizing things at the micro-level only and incurring in premature optimizations.

Reply:
Definitely will consider it for next performance related additions!

Properly measuring seems like a big task, something that we should discuss, investigate and then implement.